src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java

```diff
@@ -651,4 +698,43 @@ private DslPart createResponseForGetTaskForTaskType() {
                         .datetime("next_hearing_date", "yyyy-MM-dd'T'HH:mm:ssZ")
                 )).build();
     }
+
+    private DslPart createResponseForGetTaskForTaskState() {
+        return newJsonBody(
+            o -> o
+                .minArrayLike("tasks", 1, 1,
+                              task -> task
+                                  .stringType("id", "b1a13dca-41a5-424f-b101-c67b439549d0")
+                                  .stringType("name", "review appeal skeleton argument")
+                                  .stringType("type", "reviewAppealSkeletonArgument")
+                                  .stringType("task_state", "completed")
+                                  .stringType("task_system", "SELF")
+                                  .stringType("security_classification", "PUBLIC")
+                                  .stringType("task_title", "review appeal skeleton argument")
+                                  .datetime("due_date", "yyyy-MM-dd'T'HH:mm:ssZ")
+                                  .datetime("created_date", "yyyy-MM-dd'T'HH:mm:ssZ")
+                                  .stringType("assignee", "10bac6bf-80a7-4c81-b2db-516aba826be6")
+                                  .booleanType("auto_assigned", true)
+                                  .stringType("execution_type", "Case Management Task")
+                                  .stringType("jurisdiction", "WA")
+                                  .stringType("region", "1")
+                                  .stringType("location", "765324")
+                                  .stringType("location_name", "Taylor House")
+                                  .stringType("case_type_id", "WaCaseType")
+                                  .stringType("case_id", "1617708245335399")
+                                  .stringType("case_category", "Protection")
+                                  .stringType("case_name", "Bob Smith")
+                                  .booleanType("warnings", false)
+                                  .stringType("case_management_category", "Some Case Management Category")
+                                  .stringType("work_type_id", "hearing_work")
+                                  .stringType("work_type_label", "Hearing work")
+                                  .stringValue("role_category", "LEGAL_OPERATIONS")
+                                  .stringType("description", "aDescription")
+                                  .stringType("next_hearing_id", "nextHearingId")
+                                  .datetime("next_hearing_date", "yyyy-MM-dd'T'HH:mm:ssZ")
+                                  .stringMatcher("termination_process",
+                                                 "EXUI_USER_COMPLETION|EXUI_CASE-EVENT_COMPLETION",
+                                                 "EXUI_USER_COMPLETION")
```

Comment:
It's possible the `termination_process` field could be absent and this would be valid in the API contract.  E.g. historic records.

Reply:
Could we have not edited an existing json response as we are just adding an extra field.  Seems like duplication to add a new one?